### ISSUE TITLE
Dundee Identity Server Login and Mobile Client.

### ISSUE THREAD

**Issue by studently2024** [DuendeSoftware/Support#1131]
**Which version of Duende IdentityServer are you using?**

6

**Which version of .NET are you using?**

.NET 6

**Describe the bug**
We are using the Native Mobile Client and Dundee Identity Server for the login part of our application. We use the Quickstarts and it works like a charm. What we noticed was when authenticating the mobile  app with PKCE and Authorization code flow, we noticed that the sessions are still enabled in the browser. For example, If I go and clear the cache or browser history, the session gets cleared.

Isn;t that because the Quickstart Login page (On Post) using HttpContext.Signin which adds the authenication cookie ?. 

How do we handle this for Mobile clients where you dont want the session to be set up in the browser and work purely based on tokens ?

Do you have any  samples in Quick start showing how to effectively work with Mobile clients ?

**Comment by RolandGuijt** [comment]
The recommended way for mobile apps to authenticate is described [here](https://datatracker.ietf.org/doc/html/rfc8252).

The TLDR is as follows:
- The app opens the browser on the phone initiating authorization code flow by sending the authentication request to the authorization endpoint. It should be either the system browser or Chrome custom tabs on Android or equivalent on other devices to avoid tampering.
- After logging in the browser will be redirected to a special URI scheme. E.g. myapp://callback. The URL will contain the code as a query parameter.
- The special URI scheme is registered on the device. As soon as the browser sees the scheme it will open the app. The app will get the code via the query parameter.
- The app now contacts the IdP (token endpoint) to exchange the code for token(s).

The browser stores the authentication cookie coming from the IdP as normal so that SSO still works and the user doesn't have to re-enter credentials for each use of the app.

You can find an example using MAUI [here](https://github.com/DuendeSoftware/Samples/tree/main/various/clients/Maui).
Let me know if you have any questions.
